extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -699,7 +751,7 @@ an order can still be asynchronously created.
 
 For redirect payments payment amount is bound to `redirectUrl`.
 After redirect and before the actual finalization of the payment at the provider's page, the shopper is still able to change the cart's amount within the second tab.
-If shopper decides to change cart's amount within the second tab and finalize payment within the first tab, then according to payment amount validation an error
+If the shopper decides to change the cart's amount within the second tab and finalize payment within the first tab, then according to payment amount validation an error
```

Comment:
`payment amount validation` this should be a link?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/570/commits/6ab984ae4b2bd47ae037b6c3a94111f22f9b5b5d